src/topic.js

```diff
@@ -79,10 +79,26 @@ export class Topic {
     this._tags = [];
     // Credentials such as email or phone number.
     this._credentials = [];
+    this._replacements = {};
     // Message cache, sorted by message seq values, from old to new.
     this._messages = new CBuffer((a, b) => {
-      return a.seq - b.seq;
-    }, true);
+        return a.seq - b.seq;
+      }, true,
+      function(msg) {
+        return typeof msg.head != 'undefined' && msg.head && typeof msg.head['replace-seq'] != 'undefined';
+      },
```

Comment:
`msg => (msg.head && msg.head['replace-seq'])`

Reply:
The spec https://github.com/tinode/chat/blob/master/docs/API.md#pub defines it as `replace`, not `replace-seq` because it's not just a `seq`. If it's named `replace` then you can address it as `msg.head.replace`.

If you want to use a number here, please update the API doc.